### PR TITLE
Add history to Player playlist. Add Previous & History command

### PR DESCRIPTION
**Pull request by vansante**

### CODE REVIEW

**Comment by vansante** [comment]
@SvenWiltink Should be good to go, if travis agrees

**Comment by SvenWiltink** [comment]
@vansante Naisu. I'll review the code when I get home

File: api/api.go [hunk 1]
```diff
@@ -216,6 +220,22 @@ func (api *API) NextHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (api *API) PreviousHandler(w http.ResponseWriter, r *http.Request) {
+	song, err := api.player.Next()
```
Comment (SvenWiltink) [inline api/api.go]: yeah, no

Comment (vansante) [inline api/api.go]: Hehe, copy paste ftw

Comment (vansante) [inline api/api.go]: @SvenWiltink Fixed

**Comment by vansante** [comment]
The length of history probably won't get long enough to worry between restarts (restarting the bot clears it). I doubt we'll have performance problems until we have a history in the 100.000ths

**Comment by SvenWiltink** [comment]
👍

**Pull request merged**